src/main/java/net/geant/nmaas/portal/persistent/entity/DomainGroup.java

```diff
@@ -53,6 +57,9 @@ public class DomainGroup implements Serializable {
     @ManyToMany(fetch = FetchType.LAZY, mappedBy = "groups")
     private List<Domain> domains = new ArrayList<>();
 
+    @ElementCollection(fetch = FetchType.LAZY)
+    private List<UserViewAccess> accessUsers = new ArrayList<>();
```

Comment:
We would need a new migration for this as well, right?

Anyway, can we have on the Entity/database level only list of User entities instead of this new class?

Reply:
Also rename this field to "managers"